config/log4j.properties

```diff
@@ -55,6 +55,20 @@ log4j.appender.authorizerAppender.File=${kafka.logs.dir}/kafka-authorizer.log
 log4j.appender.authorizerAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.authorizerAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
 
+log4j.appender.metricsAppender=org.apache.log4j.DailyRollingFileAppender
+log4j.appender.metricsAppender.DatePattern='.'yyyy-MM-dd-HH
+log4j.appender.metricsAppender.File=${kafka.logs.dir}/BrokerMetrics.log
+log4j.appender.metricsAppender.layout=org.apache.log4j.PatternLayout
+log4j.appender.metricsAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+// define KAFKA appender to a KAFKA topic
+log4j.appender.KAFKA=org.apache.kafka.log4jappender.KafkaLog4jAppender
+log4j.appender.KAFKA.BrokerList=localhost:9092
+log4j.appender.KAFKA.Topic=BrokerMetrics
```

Comment:
Going with .BrokerMetrics.log for the file option and __BrokerMetrics topic name for now. Thanks @ewencp